cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2022-01-22T00:00:00Z
+index-state: 2022-05-18T00:00:00Z
```

Comment:
fyi you can manage `index-state` with haskell-nix `cabalProject`, perhaps that's simpler...